metaflow/plugins/argo/argo_workflows.py

```diff
@@ -479,13 +494,15 @@ def _compile_workflow_template(self):
         # (https://github.com/argoproj/argo-workflows/issues/7432) and we are forced to
         # generate container templates at the top level (in WorkflowSpec) and maintain
         # references to them within the DAGTask.
-
-        annotations = {
-            "metaflow/production_token": self.production_token,
-            "metaflow/owner": self.username,
-            "metaflow/user": "argo-workflows",
-            "metaflow/flow_name": self.flow.name,
-        }
+        annotations = self.kubernetes_annotations
```

Comment:
we would probably also want to add the same annotations to the Sensor, so similar change to `_compile_sensor` should be enough.

Reply:
done